gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,5 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
-
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
```

Comment:
Is this needed?

Reply:
It was just the removal of that extra empty line. I can undo it.